spec/requests/content_request_spec.rb

```diff
@@ -32,4 +36,35 @@
     xit "renders the mobile menu of content pages, two levels, in correct order" do
     end
   end
+
+  describe "Temporary authorisation on public site" do
+    # login to http basic auth
+    include AuthHelper
+
+    context "With the environment variable USE_BASIC_AUTH set" do
+      it "shows a basic auth dialog if the user is not already logged in with Basic Auth" do
+        cached_env_var = ENV["USE_BASIC_AUTH"]
+        ENV["USE_BASIC_AUTH"] = "true"
+
+        get "/"
+        expect(response.status).to eq(401)
+
+        ENV["USE_BASIC_AUTH"] = cached_env_var
+      end
+
+      # This test needs to pass before the PR is accepted
+      xit "does not show a basic auth dialog if the user is already logged in with Basic Auth" do
```

Comment:
Does this need to be updated to 'it'?

Reply:
Yes this test needs to be fixed. I am having trouble passing the HTTP_AUTHORIZATION token

And I had not thought to set USE_BASIC_AUTH for all the tests.

Thanks for spotting that, will fix